package.json

```diff
@@ -1,20 +1,32 @@
 {
-  "name": "vui-table",
+  "name": "d2l-table",
   "version": "0.1.0",
```

Comment:
Since this will no longer go to NPM, you can drop the `version` and set `private: true`